mlflow/R/mlflow/R/cli.R

```diff
@@ -24,11 +24,19 @@ mlflow_cli <- function(...,
 
   python <- dirname(python_bin())
   mlflow_bin <- python_mlflow_bin()
-  env <- modifyList(list(
-    PATH = paste(python, Sys.getenv("PATH"), sep = ":"),
-    MLFLOW_CONDA_HOME = python_conda_home(),
-    MLFLOW_TRACKING_URI = mlflow_get_tracking_uri()
-  ), env)
+
+  if (!is.null(python_conda_home())) {
+    env <- modifyList(list(
+      PATH = paste(python, Sys.getenv("PATH"), sep = ":"),
+      MLFLOW_CONDA_HOME = python_conda_home(),
+      MLFLOW_TRACKING_URI = mlflow_get_tracking_uri()
+    ), env)
+  } else {
+    env <- modifyList(list(
```

Comment:
I believe that in case the python binary is not the default python you still need to put it at the start of the PATH.